ext/nginx.conf

```diff
@@ -14,20 +14,22 @@ server {
 	# ssl_certificate     /etc/letsencrypt/live/varkaria.cyou/fullchain.pem;
 	# ssl_certificate_key /etc/letsencrypt/live/varkaria.cyou/privkey.pem;
 
-	# gulag
+	# bancho.py
 	location ~^\/(?:web|api|users|ss|d|p|beatmaps|beatmapsets|community|difficulty-rating)(?:\/.*|$) {
 		proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
 		proxy_set_header X-Real-IP $remote_addr;
 		proxy_set_header Host $http_host;
+		add_header Access-Control-Allow-Origin *; 
 		proxy_redirect off;
 		proxy_pass http://bancho;
 	}
 
-	# gulag-web
+	# guweb
 	location / {
 		proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
 		proxy_set_header X-Real-IP $remote_addr;
 		proxy_set_header Host $http_host;
+		add_header Access-Control-Allow-Origin *; 
```

Comment:
i think this isn't required for guweb?

Reply:
iirc you get errors when trying to load stuff on the website otherwise

Reply:
well i understand for the API calls to bancho.py, but not sure what content would be required here in guweb. perhaps?